Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -327,32 +333,32 @@ private IEnumerable<ObjectInitExpr> InitRequest()
             private MethodDeclarationSyntax Sync(string methodName, IEnumerable<Typ> snippetTyps, object initRequest, object makeRequest) =>
                 Method(Public, VoidType, methodName)()
                     .WithBody(
-                        $"// Snippet: {Method.SyncMethodName}({SnippetTypes(snippetTyps)}{nameof(CallSettings)})",
+                        IncludeDocMarkers ? $"// Snippet: {Method.SyncMethodName}({SnippetTypes(snippetTyps)}{nameof(CallSettings)})" : null,
```

Comment:
Just as a thought, perhaps a method of `DocMarker(string value) => IncludeDocMarker ? value : null;` would simplify this code a bit? Could definitely be done as a follow-up though. (Just having the same logic many, many times bothers me a bit. I can see us having to change them all again, where with a suitable abstraction we could change one thing.)